packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -316,6 +316,7 @@ export const studioLocaleStrings = defineLocalesResources('studio', {
 
   /** --- Configuration issues --- */
 
+  'configuration-issues.button.label': 'Configuration issues',
```

Comment:
don't forget a comment above this describing the purpose. This is used by a process to help translate these strings to other languages.